client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -16,7 +16,8 @@ import {
 	getImageEditorTransform,
 	getImageEditorFileInfo,
 	getImageEditorCrop,
-	isImageEditorImageLoaded
+	isImageEditorImageLoaded,
+	getImageMeetsMinimumDimensions
```

Comment:
h/t @southp whose idea I think it was to be able to access the store from the console in the browser